sherpa/astro/xspec/__init__.py

```diff
@@ -502,7 +507,7 @@ class XSapec(XSAdditiveModel):
 
     """
 
-    _calc = _xspec.xsaped
+    __function__ = "xsaped"
```

Comment:
It does if the model is enabled, because in that case `_calc` is exactly what it used to be. If the model is not enabled by a decorator then the name is arbitrarily set to the name of handler function we are using in the implementation, at this point 'wrapped'.